src/test/scala/com/databricks/labs/overwatch/pipeline/InitializeTest.scala

```diff
@@ -7,18 +7,20 @@ import com.databricks.labs.overwatch.utils.{BadConfigException, Config, Overwatc
 import com.fasterxml.jackson.core.JsonParseException
 import com.fasterxml.jackson.core.io.JsonEOFException
 import com.github.mrpowers.spark.fast.tests.DataFrameComparer
-import org.scalatest.PrivateMethodTester
+import org.scalatest.{Ignore, PrivateMethodTester}
 import org.scalatest.funspec.AnyFunSpec
 
+
 class InitializeTest extends AnyFunSpec with DataFrameComparer with SparkSessionTestWrapper with PrivateMethodTester {
   describe("Tests for Initializer.isPVC") {
     it ("should validate isPVC as false when org id if doesn't have ilb") {
       val conf = new Config
       conf.setOrganizationId("demo")
+      conf.setDeploymentType("default")
       val init = new Initializer(conf, disableValidations = false, isSnap = false, initDB = true)
-
-      val isPVC = PrivateMethod[Boolean]('isPVC)
-      val actual = init.Init invokePrivate isPVC()
+//      val isPVC = PrivateMethod[Boolean]('isPVC)
+//      val actual = init.Init invokePrivate isPVC()
```

Comment:
remove this commented line if not needed.

Commented out lines have been left throughout this file -- please remove all your commented lines that are no longer needed in this file.